dotnet/src/Microsoft.Agents.AI.Workflows/AgentWorkflowBuilder.cs

```diff
@@ -124,8 +124,8 @@ private static Workflow BuildConcurrentCore(
         // so that the final accumulator receives a single list of messages from each agent. Otherwise, the
         // accumulator would not be able to determine what came from what agent, as there's currently no
         // provenance tracking exposed in the workflow context passed to a handler.
-        ExecutorIsh[] agentExecutors = (from agent in agents select (ExecutorIsh)new AgentRunStreamingExecutor(agent, includeInputInOutput: false)).ToArray();
-        ExecutorIsh[] accumulators = [.. from agent in agentExecutors select (ExecutorIsh)new CollectChatMessagesExecutor($"Batcher/{agent.Id}")];
+        ExecutorRegistration[] agentExecutors = (from agent in agents select (ExecutorRegistration)new AgentRunStreamingExecutor(agent, includeInputInOutput: false)).ToArray();
```

Comment:
"Executors"... these are now "Registrations"?

Reply:
I am with @stephentoub how about WorkflowExecutor or a similar name? thus adding parity to the naming in Python?